bgpd/bgp_nht.c

```diff
@@ -212,22 +212,66 @@ void bgp_replace_nexthop_by_peer(struct peer *from, struct peer *to)
 		bnct->nht_info = to;
 }
 
+/*
+ * Returns the bnc whose bnc->nht_info matches the LL peer by
+ * looping through the IPv6 nexthop table
+ */
+static struct bgp_nexthop_cache *
+bgp_find_nexthop_matching_peer(struct peer *peer)
```

Comment:
nit: just a functions naming could be "improved" like `bgp_find_ipv6_nexthop_matching_peer`?

Reply:
fixed